### PR TITLE
[7274] replace django-ckeditor with django-ckeditor-5

### PR DESCRIPTION
**Pull request by goapunk**
needs https://github.com/liqd/adhocracy4/pull/1499

- [ ] find solution to remove old django-ckeditor (broken migrations when removing the package) in a new PR

### CODE REVIEW

File: meinberlin/apps/activities/migrations/0004_alter_activity_description.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+        ("meinberlin_activities", "0003_add_helptext"),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name="activity",
+            name="description",
+            field=django_ckeditor_5.fields.CKEditor5Field(verbose_name="Description"),
+        ),
+    ]
```
Comment (m4ra) [inline meinberlin/apps/activities/migrations/0004_alter_activity_description.py:18]: maybe first add a new field, e.g `description_ck5`, then do the data migration from (old) `description` to new one, then drop `description` field, and in a new migration file, rename `description_ck5` to `description`

**Review by m4ra — COMMENTED** [review]
See my comment for how to deal with migrating date from old field to new ckeditor5.

File: meinberlin/apps/activities/migrations/0004_alter_activity_description.py (hunk 1 above)
Comment (goapunk) [reply meinberlin/apps/activities/migrations/0004_alter_activity_description.py:18]: This migration works fine already I think. I tagged you on the a4 pr as I don't really know what to do there.

**Review by m4ra — APPROVED** [review]

**Review by philli-m — CHANGES_REQUESTED** [review]
tested functionality and following errors:
you cannot save a video
when you add an image you get a browser error and img disappears again error:
Couldn't upload file: automne07-56386274.jpg.
weirdly managed to add 1 logo image and another graphic, seems to be an issue with photos
no text wrapping options around images

**Comment by goapunk** [comment]
> tested functionality and following errors: you cannot save a video when you add an image you get a browser error and img disappears again error: Couldn't upload file: automne07-56386274.jpg. weirdly managed to add 1 logo image and another graphic, seems to be an issue with photos no text wrapping options around images

the issues should be fixed now

**Review by philli-m — APPROVED** [review]
approved, just need to consult with Caro concerning the old video issue

**Review by philli-m — APPROVED** [review]
videos still there so looking good! will give this to testers now

**Pull request merged**